src/runner.ts

```diff
@@ -31,7 +31,8 @@ export class TestRunner {
     private readonly concurrency: ConfigValue<number>,
     private readonly nodejsPath: ConfigValue<string>,
     extensionDir: string,
-    private readonly nodejsParameters: ConfigValue<string[]>
+    private readonly nodejsParameters: ConfigValue<string[]>,
+    public readonly extensions: ConfigValue<any[]>
```

Comment:
```suggestion
    private readonly extensions: ConfigValue<any[]>
```

^ since these should get passed into the Controller constructor

Reply:
Done